# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1546826319 28800
#      Sun Jan 06 17:58:39 2019 -0800
# Branch stable
# Node ID c69b7519d7ebf481f5322fdf3aa3b42a7dc9494b
# Parent  a08602475855f54fa94b7472efa39ae3cfb7cc03
build: populate $(OCT_FILES) to restore 'make uninstall' functionality

* libgui/graphics/module.mk (OCT_FILES): Append $(GRAPHICS_OCT_FILES) to the
list.
* libinterp/dldfcn/config-module.awk (OCT_FILES): Append $(DLDFCN_OCT_FILES)
to the list.

diff --git a/libgui/graphics/module.mk b/libgui/graphics/module.mk
--- a/libgui/graphics/module.mk
+++ b/libgui/graphics/module.mk
@@ -169,16 +169,18 @@ octlib_LTLIBRARIES += $(LIBOCTGUI_GRAPHI
 GRAPHICS_DEFUN_FILES = %reldir%/__init_qt__.cc
 
 GRAPHICS_OCT_FILES = $(LIBOCTGUI_GRAPHICS_LIB:.la=.oct)
 
 OCTAVE_INTERPRETER_TARGETS += $(GRAPHICS_OCT_FILES)
 
 OCT_FILE_LIBS += $(LIBOCTGUI_GRAPHICS_LIB)
 
+OCT_FILES += $(GRAPHICS_OCT_FILES)
+
 ## Use stamp files to avoid problems with checking timestamps
 ## of symbolic links
 
 %reldir%/__init_qt__.oct : $(LIBOCTGUI_GRAPHICS_LIB)
 	$(AM_V_GEN)$(INSTALL_PROGRAM) %reldir%/.libs/$(shell $(SED) -n -e "s/dlname='\([^']*\)'/\1/p" < $<) $@
 
 GRAPHICS_PKG_ADD_FILE = %reldir%/PKG_ADD
 
diff --git a/libinterp/dldfcn/config-module.awk b/libinterp/dldfcn/config-module.awk
--- a/libinterp/dldfcn/config-module.awk
+++ b/libinterp/dldfcn/config-module.awk
@@ -99,16 +99,19 @@ BEGIN {
   print "  $(DLDFCN_PKG_ADD_FILE)";
   print "";
   print "OCTAVE_INTERPRETER_TARGETS += \\";
   print "  $(DLDFCN_OCT_FILES)";
   print "";
   print "OCT_FILE_LIBS += \\";
   print " $(DLDFCN_LIBS)";
   print "";
+  print "OCT_FILES += \\";
+  print " $(DLDFCN_OCT_FILES)";
+  print "";
   print "DIRSTAMP_FILES += %reldir%/$(octave_dirstamp)";
 
   print "";
   print "libinterp_CLEANFILES += \\";
   print "  $(DLDFCN_PKG_ADD_FILE) \\";
   print "  $(DLDFCN_OCT_FILES)";
   print "";
   print "libinterp_MAINTAINERCLEANFILES += \\";
